### PR TITLE
CODEOWNERS: some more rules

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

I missed #27079. It'll help me to deal with GitHub notifications...

### CODE REVIEW

**Comment by fabpot** [comment]
2.7 is not maintained anymore :)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 /src/Symfony/Component/Ldap/* @csarrazi
 # Lock
 /src/Symfony/Component/Lock/* @jderusse
 # Messenger
 /src/Symfony/Bridge/Doctrine/Messenger/* @sroze
 /src/Symfony/Component/Messenger/* @sroze
+# PropertyInfo
+/src/Symfony/Component/PropertyInfo/* @dunglas
+/src/Symfony/Bridge/Doctrine/PropertyInfo/* @dunglas
+# Serializer
```
Comment (fabpot) [inline .github/CODEOWNERS]: Should be LDAP, right?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (dunglas) [reply .github/CODEOWNERS]: Faulty copy/paste ^^ (should be Serializer). Fixed.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request merged**